content/automate/ManageIQ/System/Event/EmsEvent/RHEVM.class/__methods__/update_vm_import_status.rb

```diff
@@ -0,0 +1,13 @@
+#
```

Comment:
@matobet, Here's a spec test I created for the event class method. It needs work, but it has the basics. Also, I had to modify the Automate method for the spec tests to run properly, it probably needs work/testing as well. I hope this helps.
`require_domain_file

describe ManageIQ::Automate::System::Event::EmsEvent::
          RHEVM::UpdateVmImportStatus do

  let(:user)          { FactoryGirl.create(:user_with_group) }
  let(:miq_server)    { EvmSpecHelper.local_miq_server }
  let(:ems)           { FactoryGirl.create(:ems_vmware_with_authentication, :zone => FactoryGirl.create(:zone)) }
  let(:vm)            { FactoryGirl.create(:vm_vmware, :ext_management_system => ems) }
  let(:miq_event)     { FactoryGirl.create(:miq_event, :event_type => "some_event") }

  let(:ems_event) do
    FactoryGirl.create(:ems_event, :vm_or_template => vm, :ext_management_system => ems)
  end
  let(:svc_model_miq_server) { MiqAeMethodService::MiqAeServiceMiqServer.find(miq_server.id) }
  let(:svc_model_user) { MiqAeMethodService::MiqAeServiceUser.find(user.id) }

  let(:root_hash) do
    { 'event_stream' => MiqAeMethodService::MiqAeServiceEmsEvent.find(ems_event.id),
      'event_type'   => 'IMPORTEXPORT_IMPORT_VM_FAILED' }
  end

  let(:ae_service) do
    Spec::Support::MiqAeMockService.new(root_object).tap do |service|
      current_object = Spec::Support::MiqAeMockObject.new
      current_object.parent = root_object
      service.object = current_object
    end
  end

  let(:root_object) do
    obj = Spec::Support::MiqAeMockObject.new(root_hash)
    obj["user"] = svc_model_user
    obj["miq_server"] = svc_model_miq_server
    obj
  end

  it "method succeeds" do
    described_class.new(ae_service).main
  end
end



`